docs/docs/models-definition.md

```diff
@@ -1,25 +1,30 @@
 ## Definition
 
-To define mappings between a model and a table, use the `define` method. Sequelize will then automatically add the attributes `createdAt` and `updatedAt` to it. So you will be able to know when the database entry went into the db and when it was updated the last time. If you do not want timestamps on your models, only want some timestamps, or you are working with an existing database where the columns are named something else, jump straight on to [configuration ][0]to see how to do that.
+To define mappings between a model and a table in v4, you define a class that inherits from [`Sequelize.Model`](../api/model) and call [`Model.init(attributes, options)`](../api/model#init). The model name is determined by the class name. You need to pass the sequelize instance as `options.sequelize` (see example)
+Sequelize will then automatically add the attributes `createdAt` and `updatedAt` to it. So you will be able to know when the database entry went into the db and when it was updated the last time. If you do not want timestamps on your models, only want some timestamps, or you are working with an existing database where the columns are named something else, jump straight on to [configuration ][0]to see how to do that.
 
+Defining a model with [`sequelize.define()`](../api/sequelize#define) as in previous versions is still supported. Check the v3 docs for examples.
 
 ```js
-var Project = sequelize.define('project', {
+class Project extends Sequelize.Model { }
+Project.init({
   title: Sequelize.STRING,
   description: Sequelize.TEXT
-})
+}, { sequelize })
```

Comment:
> Because the decorators can normalize their input before decorating the class, while you can't if you are declaring statically for the reasons already mentioned. I admit though that from my 3 example it is definitely the most ugly one. If I wasn't using TS I would just go with init().

[Mobx](https://github.com/mobxjs/mobx) has great decorator syntax, that's ES6 / Babel compatible.

I have code snippets like this working in production:

```
import { observable } from 'mobx';

export default class Menu {
  @observable active = false;
}
```

... which is available in Babel using the [transform-decorators-legacy](https://github.com/loganfsmyth/babel-plugin-transform-decorators-legacy) plug-in.

Perhaps there's a comparable syntax? Something like:

```
@options({ tableName: 'user', sequelize }) class User extends Model {
  @attribute name = {
    type: DataTypes.STRING,
    unique: true,
    // ...
  }
}
```

BTW, I wound up here after digging around for v4 documentation and bouncing between posts referencing `Model.init()` vs `define()`... has this been resolved yet? Would really like not to go back to v3 if I can help it... too much ES6 goodness here already :-)